perun-core/src/main/java/cz/metacentrum/perun/core/blImpl/UsersManagerBlImpl.java

```diff
@@ -1277,7 +1278,25 @@ public void validatePasswordAndSetExtSources(PerunSession sess, User user, Strin
 				} catch(UserExtSourceExistsException ex) {
 					//this is OK
 				}
-
+			} else if (loginNamespace.equals("dummy")) {
```

Comment:
Huh? For testing purposes I suppose. Could you please add a comment here at least?